vignettes/parallel.Rmd

```diff
@@ -99,13 +102,16 @@ Multisession-based:
 ```{r}
 options(pizzarr.parallel_read_enabled = "future")
 
+oldplan <- future::plan()
 future::plan(future::multisession, workers = 4)
 
 root <- pizzarr_sample("dog.ome.zarr")
 store <- SlowDirectoryStore$new(root)
 zarr_arr <- zarr_open(store = store, path = "/0")
 arr <- zarr_arr$get_item("...")$data
 sum(arr)
+
+future::plan(oldplan)
```

Comment:
CRAN and others see any change of options of other session state like `par()` as things that need to be cleaned up after running code.

Reply:
I see. I also see we might need to use `on.exit` in certain functions to ensure the cleanup happens

Reply:
Yep.